site/docs/releases.md

```diff
@@ -71,6 +71,88 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
+    - Add max allowed failed commits to RewriteDataFiles when partial progress is enabled ([\#9611](https://github.com/apache/iceberg/pull/9611))
+    - Fix issue when partitioning by UUID ([\#8250](https://github.com/apache/iceberg/pull/8250))
+    - Use bulk deletes in rewrite manifests action ([\#10343](https://github.com/apache/iceberg/pull/10343))
+* Flink
+    - Remove Flink 1.16 support ([\#10154](https://github.com/apache/iceberg/pull/10154))
+    - Add support for Flink 1.19 ([\#10112](https://github.com/apache/iceberg/pull/10112))
+    - Apply DeleteGranularity for writes ([\#10200](https://github.com/apache/iceberg/pull/10200))
+    - Move ParquetReader to LogicalTypeAnnotationVisitor ([\#9719](https://github.com/apache/iceberg/pull/9719))
+    - Pre-create fieldGetters to avoid constructing them for each row ([\#10565](https://github.com/apache/iceberg/pull/10565))
+    - Prevent setting endTag/endSnapshotId for streaming source ([\#10207](https://github.com/apache/iceberg/pull/10207))
+    - Implement range partitioner for map data statistics ([\#9321](https://github.com/apache/iceberg/pull/9321))
+    - refactor sink shuffling statistics collection ([\#10331](https://github.com/apache/iceberg/pull/10331))
+* Hive
+    - Fix metadata file not found ([\#10069](https://github.com/apache/iceberg/pull/10069))
+    - Use base table metadata to create HiveLock ([\#10016](https://github.com/apache/iceberg/pull/10016))
+    - turn off the stats gathering when iceberg.hive.keep.stats is false ([\#10148](https://github.com/apache/iceberg/pull/10148))
+* Specs
+    - OpenAPI: TableRequirements should use union of subclasses ([\#10434](https://github.com/apache/iceberg/pull/10434))
+    - OpenAPI: Deprecate oauth/tokens endpoint ([\#10603](https://github.com/apache/iceberg/pull/10603))
+    - OpenAPI: Fix additionalProperties for SnapshotSummary ([\#9838](https://github.com/apache/iceberg/pull/9838))
+    - OpenAPI: Fix property names for stats/partition stats ([\#10662](https://github.com/apache/iceberg/pull/10662))
+* Vendor Integrations
+    - AWS: Make sure Signer + User Agent config are both applied ([\#10198](https://github.com/apache/iceberg/pull/10198))
+    - AWS: Retain Glue Catalog column comment after updating Iceberg table ([\#10276](https://github.com/apache/iceberg/pull/10276))
+    - AWS: Retain Glue Catalog table description after updating Iceberg table ([\#10199](https://github.com/apache/iceberg/pull/10199))
+    - AWS: Support S3 DSSE-KMS encryption ([\#8370](https://github.com/apache/iceberg/pull/8370))
+    - AWS: Close underlying executor for DynamoDb LockManager ([\#10132](https://github.com/apache/iceberg/pull/10132))
+    - AWS: Add Iceberg version to UserAgent in S3 requests ([\#9963](https://github.com/apache/iceberg/pull/9963))
+    - Azure: Make AzureProperties w/ shared-key creds serializable ([\#10045](https://github.com/apache/iceberg/pull/10045))
+
```

Comment:
should this also have a `Dependencies` section like 1.5.0? People pointed out in the past that it's quite helpful to immediately see which dependencies have been updated

Reply:
Good point. I thought dependabot updates are not super helpful, but if people said they are, I'm happy to add this.
I'm updating the PR.

Reply:
Sorry, I forgot to update this section yesterday. I'm doing it now.

Reply:
I added `Dependencies` section for key updates.